R/install.R

```diff
@@ -205,6 +205,18 @@ install_cmdstan <- function(dir = NULL,
       append = TRUE
     )
   }
+
+  # Building fails on Apple silicon with < v2.31 due to a makefiles setting
+  # for stanc3, so manually implement the patch if needed from:
+  # https://github.com/stan-dev/cmdstan/pull/1127
+  if (os_is_macos() && R.version$arch == "aarch64") {
```

Comment:
Do we want to do this in all cases, for v2.31+ as well?

Reply:
Good point, might as well to be safe